scout/adapter/mongo/omics_variant.py

```diff
@@ -180,9 +183,21 @@ def omics_variants(
         else:
             nr_of_variants = skip + nr_of_variants
 
-        query = self.build_query(case_id, query=query, category=category, build=build)
+        variants_query = self.build_query(case_id, query=query, category=category, build=build)
+
+        if query.get("sort_by") and query.get("sort_order"):
+            return (
+                self.omics_variant_collection.find(variants_query, projection)
+                .sort([(query.get("sort_by"), SORTING_ORDER[query.get("sort_order")])])
```

Comment:
👍 It's few variants so far. Let's hope we don't set a bothersome precedent. 😅

Reply:
Right? 😟